bundle/config/root.go

```diff
@@ -52,7 +52,7 @@ type Root struct {
 	Targets map[string]*Target `json:"targets,omitempty"`
 
 	// DEPRECATED. Left for backward compatibility with Targets
-	Environments map[string]*Target `json:"environments,omitempty" bundle:"deprecated"`
+	Environments map[string]*Target `json:"environments,omitempty"`
```

Comment:
Why did this lose bundle:deprecated tag?

Reply:
It seems source of truth for deprecation is annotations.yml? Was bundle:deprecated tag never used for anything?

Reply:
Originally the tag was functionally used to omit fields in the JSON schema that were deprecated. With Russell's PR, we rely on the annotations in `annotations.yml` instead, so there's no need to keep the tag around.

The doc string above marks the field as deprecated anyway.

Reply:
The diff in `libs/jsonschema/from_type.go` removes the functionality that this tag had.